salt/modules/yumpkg.py

```diff
@@ -1458,7 +1458,12 @@ def install(
 
     try:
         pkg_params, pkg_type = __salt__["pkg_resource.parse_targets"](
-            name, pkgs, sources, saltenv=saltenv, normalize=normalize, **kwargs
+            name,
+            pkgs,
+            sources,
+            saltenv=saltenv,
+            normalize=normalize and kwargs.get("split_arch", True),
```

Comment:
Can you add documentation in the functions doc string for this new kwarg.

Also, another question:

Based off the `normalize` docs it seems like this issue is already solved if you pass in `normalize=False`. Is that not the case?

Reply:
There is a section which works different than just to set `normalize=False`:
https://github.com/saltstack/salt/pull/62029/files#diff-f5522aab89c593a60a281d5d2951909c938e653205f99f9d651399421f94eb0fR2177-R2179

And the other problem is `pkg` state module, which also does the normalization, and this PR is implementing the way to avoid confusions with the resulting names. The example is in the PR description.

I will add doc string :+1: